test/tools/CodeCoverageAutomation/Start-CodeCoverageRun.ps1

```diff
@@ -275,7 +278,7 @@ try
 
         Write-LogPassThru -Message "pulling sparse repo"
         "src" | Out-File -Encoding ascii .git\info\sparse-checkout -Force
-        "assets" | Out-File -Encoding ascii .git\info\sparse-checkout -Append
+        "/assets" | Out-File -Encoding ascii .git\info\sparse-checkout -Append
```

Comment:
Can you add a comment why '/' is required for assets and not for src.

Reply:
i added it for /src as that seems to work and is more clear